doc/whatsnew/2/2.14/full.rst

```diff
@@ -31,6 +31,11 @@ Release date: TBA
 
   Closes #6624
 
+* Deleted messages that do not exists anymore in pylint now raise ``useless-option-value`` instead of ``bad-option-value``
+  which permit to distinguish between genuine typoes and old configuration that could be cleaned.
```

Comment:
```suggestion
* Deleted messages that do not exist anymore in pylint now raise ``useless-option-value`` instead of ``bad-option-value``,
  which permit to distinguish between genuine typos and old configuration that could be cleaned.
```